backend/pom.xml

```diff
@@ -38,7 +38,7 @@
         </dependency>
         <dependency>
             <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-rest-client-reactive</artifactId>
+            <artifactId>quarkus-rest-client-reactive-jackson</artifactId>
```

Comment:
Jackson is needed to deserialize responses retrieved through `rest-client-reactive`. This should have been mandatory with Quarkus 2 but for some reason it worked and they fixed the bug in Quarkus 3.